timeago-generator/src/main/java/com/github/TeamNewPipe/Utils.java

```diff
@@ -1,3 +1,5 @@
+package com.github.TeamNewPipe;
```

Comment:
The package name should be `org.schabi.newpipe.timeago_generator` I believe (also in the other `timeago-generator` classes).